.buildkite/pipelines/pointer_compression.yml

```diff
@@ -1,3 +1,423 @@
+env:
+  CI_USE_POINTER_COMPRESSION: true
```

Comment:
humm I would probably change that for `CI_FORCE_NODE_POINTER_COMPRESSION`

Reply:
a13e9b9b738bcc5c97d85dff456f251bf2e8aa12